docs/SOCIAL_MEDIA_STRATEGY.md

```
@@ -0,0 +1,50 @@
+- **Content creation**: Every morning until morning meeting
+- **Posting**: Every 2 days minimum
+
+## Platforms
+
+- **LinkedIn** — Standard posts
+- **X** — Standard posts
+- **Reddit** — Insightful, discussion-oriented posts
+
+## Accounts & Roles
+
+### Company Account
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)